packages/zimbabwe/zimbabwe.svg

```diff
@@ -0,0 +1,56 @@
+   />
+   <path  
+      name="Masvingo",
+      id="masvingo",
+      d="M 347.50861,203.81668 C 348.91223,206.96812 348.84286,210.88585 355.18835,208.37794 C 363.42071,205.48879 367.31915,215.42742 374.86771,215.33989 C 378.00143,217.18354 379.92017,220.25483 384.22739,219.90115 C 384.9726,224.80043 391.30476,225.83272 393.10709,230.22403 C 396.99848,233.34049 399.08466,238.40874 402.70679,241.02702 C 406.19263,240.57094 407.93226,244.63456 411.82648,243.18764 C 414.64978,247.26322 421.91207,245.80384 419.74622,252.55025 C 418.34153,260.93226 416.59888,269.06272 416.38635,277.75726 C 415.18234,280.24297 415.60761,283.48472 413.50644,284.71918 C 413.9704,290.42701 406.58997,295.62451 410.38654,300.5636 C 411.7808,302.6828 411.97469,304.33827 415.42636,305.60498 C 416.445,309.73363 417.54205,313.21947 421.66617,314.72754 C 426.49837,317.57927 417.40681,320.98734 415.90634,324.3302 C 399.60978,340.68847 384.29098,357.75561 367.42795,373.30383 C 364.01433,375.15189 360.58869,370.26557 356.38831,369.9429 C 353.77704,371.18501 353.67041,366.60041 350.38849,366.34189 C 351.12774,364.52435 349.27636,363.62919 347.98859,362.02069 C 348.11299,361.33712 344.84774,358.66133 345.34866,356.73923 C 341.54803,358.55391 342.82087,352.38404 338.86887,351.69781 C 337.81082,350.04076 336.90753,349.02307 335.98898,347.13654 C 334.21688,344.44726 331.90908,343.4361 330.70914,340.41467 C 331.17536,337.60892 329.39597,340.23581 327.10928,338.25406 C 325.00415,339.07178 323.15275,336.70553 323.02939,334.65308 C 318.66331,332.84811 314.23924,331.26288 309.34985,331.05206 C 310.94373,327.96848 305.92322,328.93818 305.03,326.25073 C 299.80003,327.63617 298.86442,319.48758 293.99036,318.5686 C 292.96798,315.22258 291.19224,313.4205 290.15048,310.64639 C 287.78005,309.45801 287.15119,306.01898 284.39066,304.64474 C 280.39561,300.16498 288.60149,303.43075 290.15048,298.88312 C 293.74231,293.77816 299.05447,296.81107 301.9101,300.5636 C 306.33931,303.37759 311.1956,301.17422 315.82965,302.72418 C 321.12157,300.43256 320.41995,295.23397 321.10947,290.4808 C 320.62123,287.72134 324.79597,284.62898 323.02939,283.51886 C 318.78579,281.53871 321.90381,275.53524 317.98956,272.71585 C 315.64288,270.6917 313.99434,268.52486 312.22977,266.95425 C 305.24934,266.41503 314.95948,259.28794 309.10986,258.55191 C 307.02412,255.41826 303.83796,260.80599 305.50998,256.3913 C 306.22678,253.73245 306.75532,250.61975 304.79001,249.1893 C 309.68786,249.15353 314.13729,246.8177 319.42953,247.7489 C 319.72263,246.79774 318.54416,244.60934 321.10947,243.66777 C 318.27921,238.21825 326.28324,239.60268 329.2692,240.78697 C 334.98857,242.73333 331.55287,236.61133 333.58905,233.58496 C 333.74032,228.90376 333.63195,224.82349 334.54901,220.14124 C 336.81723,217.75263 336.71067,216.78211 333.58905,216.06009 C 336.67901,212.97868 342.5294,210.37986 343.90872,207.41769 C 343.46912,205.4457 344.102,201.88479 347.50861,203.81668 z "
+   />
+   <path  
+      name="Bulawayo",
+      id="byo",
```

Comment:
Could you be consistent for the ids please? Always use full names or always use short names